### ISSUE TITLE
Slash “/” in Webhook Node name prevents execution 

### ISSUE THREAD

**Issue by sheldoncoupeheure** [n8n-io/n8n#164]
**Describe the bug**
Slash “/” in Webhook Node name prevents webhook execution. The webhook is seen as non-registered

**To Reproduce**
1. load below workflow
2. save workflow
3. make sur it is not activated
4. click on normal_webhook and execute it
5. copy test webhook url in another browser window: data gets received
6. exit this node, click on Webhook/with/slash node and execute
7. copy test webhook url in browser: it doesn’t work

**Workflow**

```json
{
  "name": "reproducible webhook bug",
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        260,
        200
      ]
    },
    {
      "parameters": {
        "path": "path"
      },
      "name": "Webhook/with/slash",
      "type": "n8n-nodes-base.webhook",
      "typeVersion": 1,
      "position": [
        420,
        330
      ]
    },
    {
      "parameters": {
        "path": "hello"
      },
      "name": "normal_webhook",
      "type": "n8n-nodes-base.webhook",
      "typeVersion": 1,
      "position": [
        480,
        130
      ]
    }
  ],
  "connections": {},
  "active": false,
  "settings": {},
  "id": "4"
}
```

**Expected behavior**
normal_webhook receives event and Webhook/with/slash also receives event

**Additional context**
see community forum thread: https://community.n8n.io/t/slash-in-webhook-node-name-prevents-execution/576

**Comment by janober** [comment]
Just FYI we are currently making changes to the webhooks which should also fix this issue.

**Comment by sheldoncoupeheure** [comment]
Hi, I am happy to hear that even though I am sorry for not contributing following our discussion on n8n forums.

Thank you for the update.

**Comment by janober** [comment]
We just released a beta version of n8n with static webhooks. For existing webhook-nodes it will keep on working like before. Meaning the webhook-URL is a combination of workflow-id and webhook-name. For newly created nodes it will by default create a UUID. The value can simply get changed. It is however important to mention that webhook-URLs are now unique so not two different workflows could define the path GET `/webhook` only one can. If two define the same and both workflows gets set to active at the same time, it will error and will not allow the second one to get activated.

The beta image: `n8nio/n8n:0.70.0-static-webhooks-beta`

Please inform us if everything new and existing works fine that we can release it asap. We did already test but the more people the better obviously.Thanks!

**Comment by janober** [comment]
Got fixed in the newly released n8n@0.74.0
